recipes-support/collision-avoidance-library/coav-control_0.0.1.bb

```diff
@@ -24,6 +24,6 @@ do_install () {
     chrpath -d ${B}/tools/coav-control/coav-control
```

Comment:
You can have a single recipe creating multiple packages. It's just a matter of setting FILES_* correctly